src/qlipperpreferencesdialog.ui

```diff
@@ -35,44 +35,13 @@
         <x>0</x>
         <y>0</y>
         <width>340</width>
-        <height>217</height>
+        <height>302</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Preferences</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
-       <item row="4" column="0" colspan="2">
```

Comment:
If you wish to reorder/normalize, please, do it as separate commit to not polute this one (which handles something other).